openedx/adg/constants.py

```diff
@@ -0,0 +1,82 @@
+"""
+Constants related to ADG.
+"""
+
+
+from django.utils.translation import ugettext_lazy as _
+
+# TODO: Move this setting to model
```

Comment:
In which model do we want to move it? And if this is intended, why don't we move it now?

Reply:
We will add a new model for business lines in next half and we will remove this setting at that time.